src/synth/drumModules/snareDrum.js

```diff
@@ -32,7 +32,7 @@ export default function (audioCtx, destination, time, { level, tone, snappy }) {
   outputVCA.amplitude.value = outputLevel;
 
   // envelopes
-  const noiseEnv = new ADGenerator(LINEAR, 0.1, 75, 0, 0.5);
+  const noiseEnv = new ADGenerator(LINEAR, 0.1, 160, 0, 0.9);
```

Comment:
I've been tweaking the snare for the last few hrs. trying a few different things.
One thing for sure is that the behavior of Tone and Snappy don't currently resemble the 808 snare.  I think I made some progress but not there yet.  My approach is to try to understand what sound / dynamics are effected by the sound module.  If you totally roll off the snappy on https://codepen.io/lacostenycoder/full/bgbQzX/  for example you'll hear there is no snare rattle tone.  So what snappy does really sounds more like it's must mixing in the White noise.   I sort of have that part worked out  I just pushed update with where I'm out.  I have some other ideas I'm gonna try.  Are you on Gitter or Slack?  Maybe we can hack at this together if you have time.  LMK